servicetalk-encoding-api/src/main/java/io/servicetalk/encoding/api/ContentCodings.java

```diff
@@ -53,15 +55,19 @@ public static ContentCodec gzipDefault() {
      * a customizable {@link ContentCodec}.
      * @return a GZIP based {@link ContentCodecBuilder} that allows building
      *          a customizable GZIP {@link ContentCodec}
+     * @deprecated API replaced by <code>io.servicetalk.encoding.netty.ContentCodings#gzip()</code>
```

Comment:
There are few more case of <code> which could be converted to {@link …} or {@code …}

Reply:
Fixed. I kept everything as `{@code ...}` because we are linking classes that aren't visible in the api module.